packages/integration-testsuite/src/apolloServerTests.ts

```diff
@@ -2912,7 +2912,7 @@ export function defineIntegrationTestSuiteApolloServerTests(
         url = (await createServer(makeServerConfig([]))).url;
         await get().expect(
           200,
-          /embeddable-sandbox.cdn.apollographql.com\/_latest\/embeddable-sandbox.umd.production.min.js/s,
+          /embeddable-sandbox.cdn.apollographql.com\/17c6b43a4068da710c3713ec72f03dcb34bbd3de\/embeddable-sandbox.umd.production.min.js/s,
```

Comment:
(my suggestion is missing an import)
```suggestion
          new RegExp(`embeddable-sandbox.cdn.apollographql.com\/${DEFAULT_EMBED_VERSION}\/embeddable-sandbox.umd.production.min.js`, 's'),
```